service/src/test/java/tds/student/web/endpoints/StudentControllerIntegrationTests.java

```diff
@@ -65,22 +67,13 @@ public void shouldReturnNotFoundWhenStudentNotFound() throws Exception {
 
     @Test
     public void shouldReturnRtsAttributeForStudent() throws Exception {
-        when(rtsService.findRtsStudentPackageAttribute("client", 1, "testName")).thenReturn(Optional.of(new RtsStudentPackageAttribute("testName", "testValue")));
+        when(rtsService.findRtsStudentPackageAttributes(any(String.class), any(int.class), any(String[].class)))
+            .thenReturn(Collections.singletonList(new RtsStudentPackageAttribute("testName", "testValue")));
 
-        http.perform(get(new URI("/students/1/rts/client/testName"))
+        http.perform(get(new URI("/students/1/rts/client?attributeName=testName"))
```

Comment:
Feels like we should test passing in multiple attributeNames as well as not providing them since it is a querystring param and therefore not required to route to that endpoint (as far as i know).  Not sure if it would provide `null` or empty array for attributeNames in the controller.